src/main/java/nl/tudelft/ewi/devhub/server/web/resources/repository/AbstractProjectResource.java

```diff
@@ -188,8 +192,8 @@ protected AbstractProjectResource(final TemplateEngine templateEngine,
 			} else throw e;
 		}
 
-		if (branchDeletion != null) {
-			parameters.put("deleteSuccessful", branchDeletion);
+		if (branchDeletionStatus != null) {
+			parameters.put("deleteStatus", branchDeletionStatus);
```

Comment:
I thought having a `Boolean` was preferable, and requires less boilerplate as well? :wink: